src/components/Footer/Footer.tsx

```diff
@@ -6,7 +6,7 @@ import { FooterData } from "../../assets/data/FooterData";
 
 export function Footer() {
   return (
-    <main className={styles.main}>
+    <main className={styles.main} id="Footer">
```

Comment:
this should be an enum in a /types.ts folder such as HomepageSection then this would be `id={HomepageSection.Footer}` and you'd reference it in `src/assets/data/MenuItemData.js` as well